src/main/groovy/com/yodle/vantage/GenerateVantageJsonTask.groovy

```diff
@@ -34,6 +34,15 @@ class GenerateVantageJsonTask extends DefaultTask implements ReportProvider
 {
   public static final String DEFAULT_PROFILE = "default"
   public static final String LATEST_VERSION = "latest"
+  def doNotResolveConfigurations =
```

Comment:
I think you should be able to dynamically check whether or not the configuration has the method via `respondsTo` (see https://stackoverflow.com/questions/1534167/in-groovy-is-there-a-way-to-check-if-an-object-has-a-given-method), and conditionally call the method.  And if the object does not have the method then you know implicitly that it's resolvable since you're in a gradle version that doesn't have unresolvable dependencies.

If worse comes to worst and that doesn't work for some reason, we should at least have a comment with a link to those release notes explaining where the list came from.